### ISSUE TITLE
Accessibility: seperate text body into seperate elements

### ISSUE THREAD

**Issue by shortwavesurfer2009** [TheBrokenRail/Combustible#1]
When using Talkback posts with line breaks are read as a single chunk of text instead of being seperated into one element per paragraph.

Here is what combustible does:
![Screenshot_20230926-143047_Combustible](https://github.com/TheBrokenRail/Combustible/assets/116814522/079cb141-f796-492d-97e0-d19597e5af40)
![Screenshot_20230926-143059_Combustible](https://github.com/TheBrokenRail/Combustible/assets/116814522/9e859921-dcbf-447e-8886-44a4cdcb2163)
![Screenshot_20230926-144334_Combustible](https://github.com/TheBrokenRail/Combustible/assets/116814522/63456d65-0b18-4dae-8265-9032dc3af64c)

And here is how Thunder does:
![Screenshot_20230926-143545_Thunder](https://github.com/TheBrokenRail/Combustible/assets/116814522/2e905f81-277d-49a3-8d23-318d3c26858b)
![Screenshot_20230926-143548_Thunder](https://github.com/TheBrokenRail/Combustible/assets/116814522/8e52a7d6-bb38-4334-9054-122b099e2400)

**Comment by TheBrokenRail** [comment]
Do you mean something like [this](https://stackoverflow.com/questions/62137148/split-paragraphs-in-textview-for-better-readback)?

If so, I'm not entirely sure I would implement this. It seems like Talkback treats each `TextView` as its own element. If my understanding is correct, the only way to split it up would be to use multiple `TextView`s. Unfortunately, I don't think [Markwon](https://github.com/noties/Markwon) (the library Combustible uses for parsing Markdown) supports this.

[Jerboa](https://github.com/dessalines/jerboa) also uses Markdown, does it also have the issue?

**Comment by shortwavesurfer2009** [comment]
> Do you mean something like [this](https://stackoverflow.com/questions/62137148/split-paragraphs-in-textview-for-better-readback)?
> 
> If so, I'm not entirely sure I would implement this. It seems like Talkback treats each `TextView` as its own element. If my understanding is correct, the only way to split it up would be to use multiple `TextView`s. Unfortunately, I don't think [Markwon](https://github.com/noties/Markwon) (the library Combustible uses for parsing Markdown) supports this.
> 
> [Jerboa](https://github.com/dessalines/jerboa) also uses Markdown, does it also have the issue?

Yeah, that is what i mean. I havrnt used Jerboa in a long time so i had to check, and yes, it has the same issue
